apps/demo-ui/run-ui.sh

```diff
@@ -66,6 +70,31 @@ while ! curl -k "${BASEURL}/healthz" >/dev/null 2>&1; do
     sleep 1
 done
 
+# Only in cloud service, run token-protected UI proxy...
+if [[ ${SSL} != 0 && -f ${ARYN_ETC}/hostcert.pem ]]; then
+    TOKEN=$(openssl rand -hex 24)
+    # In could service, we can sudo without password...
```

Comment:
remove references to cloud service in sycamore repo?

Reply:
idk, probably a question for someone else. Is there a way to rephrase it? I don't really care but I thought a ben or a jon might. It's maybe not a great look to be adding things to the opensource clearly to support the cloud service.